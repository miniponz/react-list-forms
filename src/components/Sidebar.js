import React from 'react';
import PropTypes from 'prop-types';

function Sidebar({ children }) {
  return (
    <nav style={{ display: 'flex', flexDirection: 'column' }}>{children}</nav>
  );
}

Sidebar.propTypes = {
  children: PropTypes.node.isRequired
};

export default Sidebar;
