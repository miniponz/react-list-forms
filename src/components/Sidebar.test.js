import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('sidebar component', () => {
  it('renders the sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <p>Stuff</p>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
