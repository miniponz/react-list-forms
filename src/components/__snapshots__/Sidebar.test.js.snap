// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar component renders the sidebar 1`] = `
<nav>
  <p>
    Stuff
  </p>
</nav>
`;
